schema/defs-linux.json

```diff
@@ -194,7 +194,8 @@
                     "type": "boolean"
                 },
                 "type": {
-                    "type": "string"
+                    "type": "string",
+                    "pattern": "^[acb]?$"
```

Comment:
We should drop the `?`, because the empty string is not a legal value.  The field is optional, so it may be *unset*.  But it cannot be an empty string (for reasons similar to our not supporting explicit `null`s, #662).